src/frontend/src/components/base/Autocomplete/Autocomplete.tsx

```diff
@@ -65,47 +73,67 @@ export function Autocomplete({
             filteredSuggestions.filterText !==
                 filteredSuggestions.items[0].name);
 
+    useEffect(() => {
+        if (filteredSuggestions.filterText === "" && value && onClearCallback)
+            onClearCallback();
+    }, [filteredSuggestions.filterText]);
```

Comment:
I wonder if it should also listen to `value` change. Or anything you noticed when developing?